sdks/go/pkg/beam/runners/universal/runnerlib/stage.go

```diff
@@ -107,10 +139,10 @@ func StageViaPortableApi(ctx context.Context, cc *grpc.ClientConn, binary, st st
 	}
 }
 
-func StageFile(filename string, stream jobpb.ArtifactStagingService_ReverseArtifactRetrievalServiceClient) error {
+func stageFile(filename string, stream jobpb.ArtifactStagingService_ReverseArtifactRetrievalServiceClient) error {
```

Comment:
I'm guessing we aren't expecting anyone to have been depending on this function being exported?

Reply:
Not in the least.

1. It's deep within the runner lib.
2. We're the only ones that depend on it in open source land.
3. It's using the fairly esoteric (WRT beam knowledge) Reverse Artifact Retreival service, which while it works, isn't something many have experience with.
4. It was brittle to begin with (erroring in ways that hid server connection closes).
5. It shouldn't have been exposed in the first place.
6. It was also undocumented, which doesn't encourage use...